### ISSUE TITLE
Note about preserving previous panic hook

### ISSUE THREAD

**Issue by Ptrskay3** [LukeMathWalker/tracing-panic#1]
Currently in the documentation the whole panic hook is replaced with a new one that emits a tracing error. Using this with Sentry for example it's possible to override [their panic hook](https://github.com/getsentry/sentry-rust/blob/bbba0e9e779c39646d12dfaea11c6ea80b478e01/sentry-panic/src/lib.rs#L68-L71) very easily depending the order you initialize them. This can cause confusion, especially for people who aren't familiar with Rust's panic machinery.

In my opinion there should be at least a mention in the documentation about playing nice with others for instance:

```rust
let next = panic::take_hook();
panic::set_hook(Box::new(move |panic_info| {
    tracing_panic::panic_hook(panic_info);
    next(panic_info);
}));
```
or maybe a helper function that does this automatically - not sure that's necessary though.

**Comment by LukeMathWalker** [comment]
I think mentioning it in the documentation is a great idea!
Would you mind opening a PR?

**Comment by Ptrskay3** [comment]
Sure thing!

**Comment by LukeMathWalker** [comment]
Released in 0.1.1, thanks!
